viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -635,6 +639,32 @@ export class Cognite3DViewer {
     return model;
   }
 
+  /**
+   * Adds a set of 360 images to the scene from the /events API in Cognite Data Fusion.
+   * @param datasource The CDF data source which holds the references to the 360 image sets.
+   * @param eventFilter The metadata filter to apply when querying events that contains the 360 images.
+   * @param setTransform An optional addition transform which will be applied to all the 360 images in this set.
+   * @example
+   * ```js
+   * const eventFilter = { site_id: "12345" };
+   * await viewer.add360ImageSet(eventFilter);
+   * ```
+   */
+  add360ImageSet(
```

Comment:
Should we declare this method async?

Reply:
It has no await, so that is not nessesary. The import thing is the return type, which is Promise<...>, so I can always add async later, and it would not be a braking change.